lib/configurators/aws/aws_test.go

```diff
@@ -706,12 +738,63 @@ func TestAWSIAMDocuments(t *testing.T) {
 				},
 			},
 		},
+		"target not in assume role": {
```

Comment:
do we have a test case target matches an `assume_role_arn`?

Reply:
in other tests yes, but I added a case for `TestAWSIAMDocuments` as well in: 0dded5313ee3750059eb4160aa2a497a14daa65c